routes/api.php

```diff
@@ -1,19 +1,16 @@
 <?php
 
+use App\Http\Controllers\CityController;
 use Illuminate\Http\Request;
 use Illuminate\Support\Facades\Route;
 
-/*
-|--------------------------------------------------------------------------
-| API Routes
-|--------------------------------------------------------------------------
-|
-| Here is where you can register API routes for your application. These
-| routes are loaded by the RouteServiceProvider and all of them will
-| be assigned to the "api" middleware group. Make something great!
-|
-*/
-
 Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
     return $request->user();
 });
+
+Route::group(['prefix' => 'cities'], function () {
+    Route::post('/', [CityController::class, 'store'])->name('cities.store');
+    Route::delete('/{city}', [CityController::class, 'destroy'])->name('cities.destroy');
+    Route::put('/{city}', [CityController::class, 'update'])->name('cities.update');
+    Route::get('/json', [CityController::class, 'getCities'])->name('cities.json');
```

Comment:
We use GET '/' (remove the 'json'), if you want to get just 1 you use GET '/{id}'